src/array/indexOf.js

```diff
@@ -0,0 +1,29 @@
+const indexOf = (searchArray, searchValue, searchAt) => {
+    if (searchArray.length - 1 < searchAt) {
+        return -1;
+    } else if (
+        searchArray.includes(searchValue) === false ||
+        (searchValue === undefined && searchAt === undefined)
+    ) {
+        return -1;
+    }
+
+    if (searchArray.includes(searchValue)) {
+        if (searchAt === undefined) {
+            for (let i = 0; i < searchArray.length; i++) {
+                if (searchArray[i] === searchValue) {
+                    return i;
+                }
+            }
+        }
+        if (searchAt !== undefined) {
```

Comment:
These two if statements are repetitive and the logic can be condensed. Anytime that I see the repetitive code it usually means that I can use a helper function to help generalize the tasks at hand. 
(in this case the `if (searchAt === undefined)` would not be needed if you just passed in a default value for your params)